### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by wipfli**
This pull requests makes the tests work with maplibre-gl-js ```v1.14.0``` and includes the tool ```gh-pages``` into the development dependencies in ```package.json```.

Thanks to @petr-pokorny-1 for getting release automation correctly set up in the GL JS repo, see https://github.com/maplibre/maplibre-gl-js/issues/92.

### CODE REVIEW

**Comment by wipfli** [comment]
If the tests work we could start thinking about setting up some GitHub action which automatically builds, tests, and deploys the static site files to the ```gh-pages``` branch. This action could be triggered whenever a pull request gets merged.

File: README.md [hunk 1]
```diff
@@ -84,11 +84,22 @@ To update or add a new example, PR the relevant changes to this repo. The exampl
...
+Test that everything is fine with:
+
+```bash
+npm run test
+```
 
 Publish the build output to GitHub Pages at [https://maplibre.github.io/maplibre-gl-js-docs/](https://maplibre.github.io/maplibre-gl-js-docs/) with:
 
 ```bash
-npm install -g gh-pages
```
Comment (nyurik) [inline README.md:92]: why is this no longer needed?

**Review by nyurik — COMMENTED** [review]
one inline comment, and one confusion -- why is there a `yarn.lock`, but all instructions use `npm` instead of `yarn`?   I would prefer to be consistent and use `yarn` everywhere since that's what seems to be the standard for the project?

File: README.md (hunk 1 above)
Comment (wipfli) [reply README.md:92]: It is in package.json now under the dev dependencies with commit 4c6fbdb.

**Comment by wipfli** [comment]
I agree it is confusing that there are ```npm``` and ```yarn``` commands. Mapbox did not seem to have a clear strategy for using one or the other. For example, at the time of forking the ```prebuild``` command uses ```npm```

https://github.com/maplibre/maplibre-gl-js-docs/blob/533acd5385be25a4b82b2cf458c2ce5b579911ac/package.json#L80

while the ```pretest``` command uses ```yarn```.

https://github.com/maplibre/maplibre-gl-js-docs/blob/533acd5385be25a4b82b2cf458c2ce5b579911ac/package.json#L84

Is one of them better than the other?

File: README.md (hunk 1 above)
Comment (wipfli) [reply README.md:92]: I installed it with 

```bash
npm install gh-pages --save-dev
```

Was this maybe wrong and I should better have used the following?

```bash
yarn add gh-pages --dev
```

**Comment by wipfli** [comment]
It is good that you pointed this inconsistency out @nyurik. In fact, this seems to be some historic burden, see https://github.com/mapbox/mapbox-gl-js-docs/pull/60 and https://github.com/mapbox/mapbox-gl-js-docs/issues/472. I suggest we do the same and switch to ```npm```-only now.

**Comment by wipfli** [comment]
Want to refile this pull request together with some updates that remove ```yarn```.

**Comment by wipfli** [comment]
switch to npm: https://github.com/mapbox/mapbox-gl-js-docs/commit/eff9be93748109890d09e64ddbe746150270f3b5?branch=eff9be93748109890d09e64ddbe746150270f3b5&diff=unified

File: README.md (hunk 1 above)
Comment (nyurik) [reply README.md:92]: ah, thx, makes sense. Perhaps we should switch to yarn in a separate PR?

**Comment by nyurik** [comment]
> switch to npm: [mapbox/mapbox-gl-js-docs@eff9be9?branch=eff9be93748109890d09e64ddbe746150270f3b5&diff=unified](https://github.com/mapbox/mapbox-gl-js-docs/commit/eff9be93748109890d09e64ddbe746150270f3b5?branch=eff9be93748109890d09e64ddbe746150270f3b5&diff=unified)

Thanks for the research, I think we should just apply that patch to this repo.  Just create a new branch from our main and cherry-pick that change to it, and create a PR from it.

**Pull request closed without merging**